### PR TITLE
[API] Project /images endpoint broken  

### PR DESCRIPTION
**Pull request by kongtiaowang**
## Brief summary of changes
Fixing 500 error. 
remove "file" column from projectimagesrow, since #6516.
"PHP Fatal error: Uncaught TypeError: basename() expects parameter 1 to be string, null given in /var/www/Loris/modules/api/php/models/projectimagesrow.class.inc:79"

#### Testing instructions (if applicable)
Test  "/api/v0.0.3/projects/Rye/images" on a loris instence with Rasinbread data set .
It will show the result.
屏幕快照 2020-06-03 11 31 13

#### Link(s) to related issue(s)
https://github.com/aces/Loris/issues/6572

### CODE REVIEW

**Comment by driusan** [comment]
Doing this reverts #6516. Why does that cause a 500 error? The internal filename is not supposed to be part of what's returned by the API.

**Comment by kongtiaowang** [comment]
@driusan  Because the file name is null, so the basename call empy value, then we got 500 error.
PHP Fatal error: Uncaught TypeError: basename() expects parameter 1 to be string, null given in /var/www/Loris/modules/api/php/models/projectimagesrow.class.inc:79\nStack trace:\n#0 /var/www/Loris/modules/api/php/models/projectimagesrow.class.inc(79): basename(NULL)\n#1 /var/www/Loris/src/Data/Table.php(84): LORIS\api\Models\ProjectImagesRow->toJSON(Object(User))\n#2

**Comment by driusan** [comment]
Makese sense, but then you need to remove the File key from the array that gets serialized to json

File: modules/api/php/models/projectimagesrow.class.inc [hunk 1]
```diff
@@ -69,18 +69,16 @@ class ProjectImagesRow implements \LORIS\Data\DataInstance
...
             'QC_status'  => $this->_qcstatus,
             'Selected'   => $this->_selected,
         );
 
-        $filename = basename($obj['File']);
         $candid   = $obj['Candidate'];
         $visit    = $obj['Visit'];
 
-        $obj['Link']       = "/candidates/$candid/$visit/images/$filename";
+        $obj['Link']       = "/candidates/$candid/$visit/images/";
```
[the remark this answers is not in the document]
Comment (kongtiaowang) [reply modules/api/php/models/projectimagesrow.class.inc:81]: @driusan It means I need to add filename into the returned by the API.

[the remark this answers is not in the document]
Comment (kongtiaowang) [reply modules/api/php/models/projectimagesrow.class.inc:81]: @driusan If I have removed the File name from obj, how should I keep filename/basename for this? File name will always be null.

[the remark this answers is not in the document]
Comment (driusan) [reply modules/api/php/models/projectimagesrow.class.inc:79]: Why can't you use `$this->_filename` (the value that was being put in the obj) without putting it in the object?

**Comment by driusan** [comment]
@HenriRabalais Can you test/review and make sure this doesn't cause regressions with your PR (#6516)?

File: modules/api/php/models/projectimagesrow.class.inc (hunk 1 above)
[the remark this answers is not in the document]
Comment (kongtiaowang) [reply modules/api/php/models/projectimagesrow.class.inc]: You are right.

**Pull request merged**